cloud-functions/src/utils/fetchTokenUSDPrice.ts

```
@@ -0,0 +1,62 @@
+				include_24hr_change: 'true',
+				vs_currencies: 'usd'
+			}
+		});
+
+		const responseData = response.data as CoinGeckoResponse;
+
+		if (
+			!responseData ||
+			typeof responseData !== 'object' ||
+			Object.keys(responseData).length === 0 ||
+			!responseData.hasOwnProperty(coinId) ||
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)